<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SubToolBar</class>
 <widget class="QWidget" name="SubToolBar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1265</width>
    <height>65</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>65</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>65</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#label_title
{
font-size: 22px;
font-family: Microsoft YaHei;
font-weight: bold;
color: #E0E0E1;
line-height: 45px;
}
#SubToolBar
{
background: #2F3443;
border:none;
border-bottom:2px solid #1269D3;
}
#pushButton_casemanage,#pushButton_design,#pushButton_rehearsal,#pushButton_navigate{
background:#313343;
border:none;
font-size: 22px;
font-family: &quot;思源黑体 CN Bold&quot;;
color: #BBC1CA;
}
#pushButton_casemanage:checked,#pushButton_design:checked,#pushButton_rehearsal:checked,#pushButton_navigate:checked{
background-color: qlineargradient(spread:pad,x1:0, y1:0, x2:0, y2:1,
stop:0 #1167CF,stop:0.5 #197CF4,stop:1 #1167CF);
border:none;
font-size: 22px;
font-family: &quot;思源黑体 CN Bold&quot;;
color: #FFFFFF;
}
#pushButton_casemanage:focus,#pushButton_design:focus,#pushButton_rehearsal:focus,#pushButton_navigate:focus{
outline:none;
}</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>20</number>
   </property>
   <property name="leftMargin">
    <number>20</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>2</number>
   </property>
   <item>
    <widget class="QLabel" name="label_title">
     <property name="minimumSize">
      <size>
       <width>70</width>
       <height>50</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>400</width>
       <height>50</height>
      </size>
     </property>
     <property name="text">
      <string>术前设计</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>62</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QPushButton" name="pushButton_casemanage">
     <property name="minimumSize">
      <size>
       <width>135</width>
       <height>63</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>135</width>
       <height>63</height>
      </size>
     </property>
     <property name="text">
      <string>病例管理</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">buttonGroup</string>
     </attribute>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pushButton_design">
     <property name="minimumSize">
      <size>
       <width>135</width>
       <height>63</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>135</width>
       <height>63</height>
      </size>
     </property>
     <property name="text">
      <string>术前设计</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">buttonGroup</string>
     </attribute>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pushButton_rehearsal">
     <property name="minimumSize">
      <size>
       <width>135</width>
       <height>63</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>135</width>
       <height>63</height>
      </size>
     </property>
     <property name="text">
      <string>方案预演</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">buttonGroup</string>
     </attribute>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pushButton_navigate">
     <property name="minimumSize">
      <size>
       <width>135</width>
       <height>63</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>135</width>
       <height>63</height>
      </size>
     </property>
     <property name="text">
      <string>手术导航</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">buttonGroup</string>
     </attribute>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../qr.qrc"/>
 </resources>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
